steward/src/config.rs

```diff
@@ -14,10 +14,10 @@ use signatory::FsKeyStore;
 use std::{net::SocketAddr, path::Path, time::Duration};
 
 lazy_static! {
-    pub static ref DELEGATE_KEY: PrivateKey = {
+    pub static ref DELEGATE_KEY: cosmos_gravity::crypto::PrivateKey = {
```

Comment:
There's no reason internally to swap to this key type. It's only necessary for gravity bridge functions that expect this type, and it's specific to feature flags for ethermint. We should use the regular `deep_space` `PrivateKey` wherever possible inside Steward.